src/OrganisationRegistry.SqlServer/Security/OrganisationTreeView.cs

```diff
@@ -149,9 +149,11 @@ public async Task Handle(DbConnection dbConnection, DbTransaction dbTransaction,
         {
             using (var context = contextFactory.CreateTransactional(dbConnection, dbTransaction))
             {
+                var contextOrganisationTreeList = context.OrganisationTreeList.ToList();
```

Comment:
Does this improve perf?

Can we use async for this?

Reply:
op z'n minst geen n+1 meer, async is perfectly possible, implemented.